app/javascript/src/components/Dashboard/Notes/NoteDetail.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Avatar, Tooltip, Dropdown, Tag } from "neetoui";
+
+import { calculateTimeDifference, dayTimeFormatter } from "utils/date";
+
+const NoteDetail = ({ title, description, created_at: createdAt }) => (
+  <div className="my-2 flex w-full flex-col rounded-sm border border-solid border-gray-300 p-4 shadow">
+    <div className="flex justify-between">
+      <Typography style="h4" weight="semibold">
+        {title}
+      </Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <div className="p-1.5">
+          <li>Edit</li>
+          <li>Delete</li>
+        </div>
+      </Dropdown>
+    </div>
+    <Typography style="body2">{description}</Typography>
+    <div className="h-0 border-b border-solid border-gray-300" />
```

Comment:
Why do we need this div here?

Reply:
I used it instead of horizontal rule(hr tag).

Reply:
@Ishaan-Singhal What advantage do you get by using a div here over the semantic `hr`?

Reply:
There was no specific reason for using div. I thought it was only a presentational change. I will replace it with an hr tag.

Reply:
@Ishaan-Singhal Read: https://www.thoughtco.com/why-use-semantic-html-3468271

Reply:
The articles were really insightful. But I have a doubt, for example, how to decide when to prefer semantic tags such as h1,h2 or p tags and when to use Typography available in neetoUI?

Reply:
@Ishaan-Singhal Always use the dedicated neetoUI tags, if present. If such a tag is not present, then make the second choice as the vanilla semantic HTML tags. 

> how to decide when to prefer semantic tags such as h1,h2 or p tags and when to use Typography available in neetoUI?

Use the `Typography` tag with the appropriate `style` and `component`. Refer: https://neetoui.onrender.com/?path=/docs/foundation-typography--heading-1. neetoUI tags use semantic html underneath the hood.

PS: Here you should be using an `hr` tag.

Reply:
Thanks for the explanation. I will keep this in mind.